src/OrchardCore/OrchardCore.ReCaptcha.Core/Configuration/ReCaptchaSettingsConfiguration.cs

```diff
@@ -14,9 +14,9 @@ public ReCaptchaSettingsConfiguration(ISiteService site)
 
         public void Configure(ReCaptchaSettings options)
         {
-            var settings = _site.GetSiteSettingsAsync()
-                .GetAwaiter().GetResult()
-                .As<ReCaptchaSettings>();
+            var settings = _site.GetSettingsAsync<ReCaptchaSettings>()
+                .GetAwaiter()
+                .GetResult();
```

Comment:
How the format works this time :)

Reply:
@hishamco did you try it out in VS?

Reply:
I tried the `main` branch moving a method call into another line adds 4 space, also using ternary condition operator too

Reply:
Does it place it on the same level as this PR does?

Reply:
I faced an issue with IIS Express once it fixed I will try it in this PR

Reply:
@hishamco I do not want to want too much on this PR. Every merge will cause a conflict :) So please either approve or merge this PR once you finish what every you need to check.

Reply:
another conflict today @hishamco? Are you really checking this spacing issue? I am asking this because it seems to be a time wasting step.

Reply:
Mike, I'm not sure why you are in a rush in almost all the PRs :) as you know the time zones are different and some of us working on other PRs, issues or something else

Believe me it happen in almost OSS, I will start checking you PR today

Reply:
Because of too many conflicts. + I don't see a value is blocking a PR because you do not agree with VS formatting :) Let's focus on valuable feedback so we are not spending too much time debating or resolving conflicts that can be simply avoided. I hope you understand my point.